ivy/functional/frontends/torch/miscellaneous_ops.py

```diff
@@ -154,3 +154,8 @@ def logcumsumexp(input, dim, *, out=None):
 
 def repeat_interleave(input, repeats, dim=None, *, output_size=None):
     return ivy.repeat(input, repeats, axis=dim)
+
+
+def ravel(input):
+    reshape_shape = (int(ivy.prod(ivy.array(tuple(input.shape)))), 1)
+    return ivy.array(input, dtype=input.dtype).reshape(reshape_shape)

```

Comment:
I see what you're done here. But there is a quicker way:
`ivy.reshape(input, (-1,))` should just return the flattened array any way.

Reply:
Yes that does make it much cleaner ! 
I made that change, i think the tests pass now